lingetic-nextjs-frontend/__tests__/components/questions/FillInTheBlanks/FillInTheBlanks.test.tsx

```
@@ -14,219 +14,233 @@ import type {
+
+const ComponentWithChangeQuestionBtn = () => {
+  const [currentQuestion, setCurrentQuestion] = useState(mockQuestion);
+
+  return (
+    <>
+      <FillInTheBlanks question={currentQuestion} />
+      <button
+        onClick={() =>
+          setCurrentQuestion({ ...mockQuestion, id: "different-id" })
+        }
+      >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)